.github/workflows/zoningtaxlots_dataloading.yml

```diff
@@ -11,6 +11,11 @@ on:
         description: "Build Pluto (minor) after dataloading is complete"
         type: boolean
         default: false
+      promote_ztl:
+        description: "Promote  Zoning Tax Lots to Draft folder after build?"
```

Comment:
nit: `Promote  Zoning` should be `Promote Zoning`

Reply:
I also think this should default to `true` - just one less click for GIS